roles/ovirt-mac-pools/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: set mac pool
+  ovirt_mac_pools:
+    auth: "{{ ovirt_auth }}"
+    name: "{{ mac_pool_name | default('Default') }}"
```

Comment:
in current flow if I understand correctly, if user don't specify anything. the `ovirt-infra` role will try to remove 'ranges' from 'Default' mac-pool, I think it's not what you want, or is it?